docs/api.md

```diff
@@ -51,7 +51,7 @@ Initializes the interface to the LCD screen, and specifies the dimensions (width
 #### Syntax
 
 ```
-lcd.begin(cols, rows)
+lcd.begin(cols, rows [, charsize[=LCD_5x10DOTS]])
```

Comment:
```suggestion
lcd.begin(cols, rows)
lcd.begin(cols, rows, charsize)
```

In order to make the documentation friendly to beginners, each of the supported syntaxes should be listed. This convention is demonstrated in the constructor's documentation:

https://github.com/arduino-libraries/LiquidCrystal/blob/master/docs/api.md#syntax

Even though the two variants are the result of a default parameter rather than an overload in this case, the distinction is not significant to the user.

Reply:
Thanks for your guidance! I'm new to OSS contributions :)